src/main/resources/db/changelog/db.changelog-release-1.0.0.sql

```diff
@@ -42,27 +41,35 @@
 --https://postgis.net/docs/postgis_installation.html
 --CREATE EXTENSION IF NOT EXISTS 	postgis;
 
+-- changeset yilativs:1 context:common failOnError: true
 
---DROP SCHEMA IF EXISTS reports CASCADE; -- attacks, treatments
---DROP SCHEMA IF EXISTS metadata CASCADE; -- treatment types
---DROP SCHEMA IF EXISTS profile CASCADE; -- user profile related data: roles, privileges, passwords
-
-CREATE SCHEMA profile;
-CREATE SCHEMA metadata;
-CREATE SCHEMA report;
+CREATE SCHEMA profile; -- attacks, treatments, hormone test results, heart and motion activity
```

Comment:
It seems every time I run app I have to remove all three schemas, maybe we should add create schema or drop or perhaps add to liquibase ignore if already exists?

Reply:
you should do it only once the file is changed, if you re-run  it now it should work.